helpers/response.helper.js

```diff
@@ -0,0 +1,27 @@
+import { STATUS } from './constants';
```

Comment:
#### Critical
- The naming convention for this file is against BEM

Reply:
Thank you @aanchirinah for pointing that out. I have renamed the file to the prescribed convention.